foo_service.rb

```diff
@@ -0,0 +1,20 @@
+class CancelPendingDeposits < Service
+  def initialize(user:, bar:)
+    @user = user
+    @bar = bar
+  end
+
+  def call
+    foo = <<~EOS
```

Comment:
__[rubocop]__
* Useless assignment to variable - `foo`. [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)

<!-- Comment by [SideCI](https://sideci.com) -->